adapters/ttx/ttx.go

```diff
@@ -0,0 +1,77 @@
+package ttx
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"net/http"
+)
+
+type TtxAdapter struct {
+	endpoint string
+}
+
+func (a *TtxAdapter) MakeRequests(request *openrtb.BidRequest) ([]*adapters.RequestData, []error) {
+	var errs []error
+	jsonData, err := json.Marshal(request)
+
+	if err != nil {
+		errs = append(errs, err)
+		return nil, errs
+	}
+
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json;charset=utf-8")
+	reqData := []*adapters.RequestData{{
+		Method:  "POST",
+		Uri:     a.endpoint,
+		Body:    jsonData,
+		Headers: headers,
+	}}
+
+	return reqData, errs
+}
+
+func (a *TtxAdapter) MakeBids(internalRequest *openrtb.BidRequest, externalRequest *adapters.RequestData, response *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+	if response.StatusCode == http.StatusNoContent {
+		return nil, nil
+	}
+
+	if response.StatusCode == http.StatusBadRequest {
+		return nil, []error{&errortypes.BadInput{
+			Message: fmt.Sprintf("Unexpected status code: %d. Run with request.debug = 1 for more info", response.StatusCode),
+		}}
+	}
+
+	if response.StatusCode != http.StatusOK {
```

Comment:
I mean: what happens if somebody configures the Prebid server incorrectly, and the specify this URL:

```
https://ssc.33across.com/api/v1/s2s/
```

Instead of this one:

```
https://ssc.33across.com/api/v1/s2s
```

Is Prebid smart enough to handle the redirect behind the scenes?  Or, will this cause the request to error out?

Reply:
k -- we should follow up w/ Prebid to see how they recommend handling this scenario.